src/store/postgres/builder.rs

```diff
@@ -26,6 +27,7 @@ where
     event_buses: Vec<Box<dyn EventBus<A> + Send>>,
     run_migrations: bool,
     _schema: PhantomData<Schema>,
+    id_func: fn() -> Uuid,
```

Comment:
In rust you should generally use a generic over the `Fn` trait, instead of a concrete `fn` type

Reply:
In this case I would even suggest using a single function id generation trait